web_src/js/features/comp/ComboMarkdownEditor.ts

```diff
@@ -137,12 +148,14 @@ export class ComboMarkdownEditor {
       monospaceButton.setAttribute('aria-checked', String(enabled));
     });
 
-    const easymdeButton = this.container.querySelector('.markdown-switch-easymde');
-    easymdeButton.addEventListener('click', async (e) => {
-      e.preventDefault();
-      this.userPreferredEditor = 'easymde';
-      await this.switchToEasyMDE();
-    });
+    if (this.supportEasyMDE) {
+      const easymdeButton = this.container.querySelector('.markdown-switch-easymde');
+      easymdeButton.addEventListener('click', async (e) => {
```

Comment:
```suggestion
      easymdeButton.addEventListener('click', async (e: MouseEvent) => {
```

Reply:
We do not need to do anything, because we do not use any "mouse" related event properties, the default inferred type is good enough.

![image](https://github.com/user-attachments/assets/045d087b-6a90-4b4a-b0ea-45d54f959645)

![image](https://github.com/user-attachments/assets/2a70e693-75fd-49b7-bb94-c35351199fe0)

Reply:
As long as will pass typescript strict mode, I'm fine. I thought bare `e` never passes.